reporter/src/main/kotlin/utils/ReportTableModelMapper.kt

```diff
@@ -227,13 +227,7 @@ class ReportTableModelMapper(
             summaryRows.values.toList().sortedWith(compareBy({ ortResult.isExcluded(it.id) }, { it.id }))
         )
 
-        val metadata = mutableMapOf<String, String>()
-        (ortResult.data["job_parameters"] as? Map<*, *>)?.let {
-            it.entries.associateTo(metadata) { (key, value) -> key.toString() to value.toString() }
-        }
-        (ortResult.data["process_parameters"] as? Map<*, *>)?.let {
-            it.entries.associateTo(metadata) { (key, value) -> key.toString() to value.toString() }
-        }
+        val metadata = ortResult.labels.mapKeys { it.key.substringAfter(".") }
```

Comment:
I looks a bit arbitrary to swallow the part before the first `.`, and assuming that `.` as part of a label has a special meaning is not really generic. While I'd be fine to introduce the latter as a convention, I believe we should not omit parts before `.` but use them for some visual grouping as part of the reports.

Reply:
Is what I wriote above what you mean by

> Note: A reasonable improvement would be to introduce sections for labels with the same prefix.

in the commit message?

Reply:
yes, exactly. Would you be fine doing this in a future PR.

Reply:
Ok, but could you then please, in addition to the commit message, add an according `TODO` for this line?

Also, shouldn't this commit then remove the changes done to `reporter/src/funTest/assets/static-html-reporter-test-input.yml` in the first commit?

Reply:
> Also, shouldn't this commit then remove the changes done to reporter/src/funTest/assets/static-html-reporter-test-input.yml in the first commit?

There will be a different PR soon which removes the custom data. Imo it'd better fit in there.

Reply:
I added a TODO comment.

Reply:
I've turned you proposal above into an issue, see #3062 3062.